CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 2.12.0-rc.0
+
+* [FEATURE] Add PromQL query logging. #5794
+* [FEATURE] Enable and provide binaries for `mips64` / `mips64le` architectures. #5792
+* [ENHANCEMENT] Mark deleted rule's series as stale on next evaluation. #5759
+* [ENHANCEMENT] Check for duplicate label names in remote read. #5829
```

Comment:
This could also be seen as either BUGFIX or CHANGE, not sure?

Reply:
Bugfix